<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AI Battery Management</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1. bank select</string>
    </property>
   </widget>
   <widget class="QLabel" name="bank_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>531</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2. cell select</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="PlotWidget" name="graphWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>741</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>741</width>
      <height>192</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>5</width>
      <height>10</height>
     </size>
    </property>
    <column>
     <property name="text">
      <string>타임 스탬프</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>내부저항</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>scaled</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>예측치</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>741</width>
      <height>91</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>3</width>
      <height>10</height>
     </size>
    </property>
    <column>
     <property name="text">
      <string>D+1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D+2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D+3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D+4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D+5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D+6</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D+7</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D+8</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D+9</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D+10</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D+11</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D+12</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D+13</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D+14</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D+15</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D+16</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D+17</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D+18</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D+19</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D+20</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
